tests/post_training/data/wc_reference_data.yaml

```diff
@@ -11,18 +11,18 @@ tinyllama_data_aware_awq_stateful_backend_OV:
   num_int4: 94
   num_int8: 124
 tinyllama_data_aware_awq_scale_estimation_backend_OV:
-  metric_value: 0.83795
+  metric_value: 0.84048
   num_int4: 94
   num_int8: 124
 tinyllama_data_aware_awq_scale_estimation_stateful_backend_OV:
-  metric_value: 0.83795
+  metric_value: 0.84048
   num_int4: 94
   num_int8: 124
 tinyllama_int8_data_free_backend_TORCH:
   metric_value: 0.95624
   num_int4: 0
   num_int8: 312
 tinyllama_data_aware_gptq_backend_OV:
-  metric_value: 0.79797
-  num_int4: 188
+  metric_value: 0.82187
```

Comment:
@alexsu52 is it expected improvement?

Reply:
I don't wait any improvements from this changes. I would suggest to check that scales and zero_points with/without PR.

Reply:
fixed in https://github.com/openvinotoolkit/nncf/pull/2725